lang/java.md

```diff
@@ -0,0 +1,114 @@
+---
+    lang: Java
+    syntax: java
+    title: Java is a statically typed JVM based programming language.
```

Comment:
+1 for the title 😀

Reply:
@KhafiaAyyub: This needs attention too2! :eyes: